HW2/P3/P3.txt

```diff
@@ -0,0 +1,25 @@
+results
+
+Without AVX with 1 thread(s)
+1074.652832 Million Complex FMAs in 16.3464031219 seconds, 65.7424648091 million Complex FMAs / second
+Without AVX with 2 thread(s)
+1074.652832 Million Complex FMAs in 11.316598177 seconds, 94.9625333688 million Complex FMAs / second
+Without AVX with 4 thread(s)
+1074.652832 Million Complex FMAs in 10.221462965 seconds, 105.136890451 million Complex FMAs / second
+
+With AVX with 1 thread(s)
+1059.236124 Million Complex FMAs in 3.0956158638 seconds, 342.172986121 million Complex FMAs / second
+With AVX with 2 thread(s)
+1059.236124 Million Complex FMAs in 1.60934400558 seconds, 658.178810947 million Complex FMAs / second
+With AVX with 4 thread(s)
+1059.236124 Million Complex FMAs in 1.93741607666 seconds, 546.726197207 million Complex FMAs / second
+
+
+discussion
+My computer has only one CPU, that can only simulate two threads properly.
```

Comment:
Wow, my performance is 4x better than yours. what laptop are you using for the homework? I used a macbook air with i7 cpu and 8GB RAM. I am just curious.